ledger/accountdb.go

```diff
@@ -2941,6 +3009,54 @@ func accountsPutTotals(tx *sql.Tx, totals ledgercore.AccountTotals, catchpointSt
 	return err
 }
 
+func accountsOnlineRoundParams(tx *sql.Tx) (onlineRoundParamsData []ledgercore.OnlineRoundParamsData, err error) {
```

Comment:
if no round numbers returned how do we know what is the start round? Plust it needs to be ordered by a round number...

Reply:
I guess it is better to return the start round, and insure in tests that the latest = start + len(result)-1 == dbRound

Reply:
sounds good.